### DISCUSSION TITLE
Deployment Details are not working at all on all projects and all commits

### DISCUSSION THREAD

**Discussion by webtwitter** [vercel/community#3160] [Help] [answered]
### Page to Investigate

https://vercel.com/

### Steps to Reproduce

Hello, for a couple of days now, I can't see deploy details on my account nor debug errors properly when a build fails on any of my projects.
I've tried:
- logout and login
- clear cache and everything from google app bar and logged out and logged in again

Nothing works. Can you help please?
![Screenshot_6](https://github.com/vercel/community/assets/32715017/d951cfef-17e1-408b-8cf1-acdd62f01520)

**Comment by swarnava** [comment]
Hi,

What browser and operating system are you using?

**Reply by webtwitter** [reply]
Hello,

Browser is Chrome: Version 114.0.5735.199 (Official Build) (64-bit)
OS is Windows 10 Pro

Thanks!

**Reply by swarnava** [reply]
Thank you! Are you still be able to reproduce this? Do you have a screencast ? Additionally, do you have any Chrome extension which may conflict to this?

**Reply by webtwitter** [reply]
@swarnava yes it happens very often. Here's a video for you. Hope it helps. https://www.loom.com/share/9918920f62084b3cb9bae6f88f124567?sid=0b7a7137-6c9c-4fde-83f8-fe741a85cda4
Let me know if you need more details.
BTW, it's happening for all the projects not only for that one.

If it helps, I'm using the next app version for my projects.

**Reply by webtwitter** [reply]
hey @swarnava any updates on this one? Still not working... :(

**Reply by sebastianroming** [reply]
Hey @webtwitter,

I've seen you've reached out in a Support Case – thank you!
Can you try on another machine/network, e.g. your mobile phone with mobile network, to check if you're able to see the logs?

**Comment by webtwitter** [comment]
Thanks.
It works after I've added vercel on bitdefender exceptions.


Have a great day!

**Reply by sebastianroming** [reply]
Thanks for confirming - we're looking into why this happens.

**Comment by sebastianroming** [comment, marked as answer]
Bitdefender is currently blocking Vercel's Build Logs. Please add an exception to allow Vercel traffic if you're not seeing Build Logs.

**Reply by gurgel-luke** [reply]
I'm current experiencing a similar issue despite not having Bitdefender and testing on multiple devices and browsers, all yield the same result: 

![image](https://github.com/vercel/community/assets/141527891/b0a58f96-7be3-43f6-970e-dd2806dcc424)

Currently, my build fails with "Command "npm run build" exited with 1", but I cannot see any logs, errors, or warnings populating in the Deployment Details.

**Reply by amyegan** [reply]
@LukeGurgel-ux Thanks for reporting this new issue. You can find the latest updates in the incident report at https://www.vercel-status.com/incidents/y42twddjx0sf
